src/scss/_main.scss

```diff
@@ -837,12 +823,54 @@ nav {
   }
 }
 
-.log-table tr:hover,
-.log-table tr:focus {
+.logs .-focus {
+  background: var(--color-focus);
+}
+
+.line a {
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+}
+
+.logs-table {
+  display: block;
+  table-layout: fixed;
+  width: 100%;
+  height: 22rem;
+  overflow-y: auto;
+
+  min-height: 10rem;
+  max-height: 75rem;
+  padding-bottom: 1rem;
+  padding-top: 1rem;
+  resize: vertical;
+
+  scrollbar-color: var(--color-coal-light) var(--color-bg-dark);
```

Comment:
added to support firefox
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars

firefox also does not support restyling the automatic resizer :( 
https://caniuse.com/css-resize